specs/commandline/sign.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "notation sign"
+description: "The notation sign command description and usage"
+keywords: "notation, sign"
+---
+# notation sign
+## Description
+Use `notation sign` to sign artifacts.
+
+If the signing artifact is a container image stored in a registry, the signature is pushed to the registry by default after signing successfully, and the digest of the container image is returned.
+## Outline
+```console
+$ notation sign --help
+Sign artifacts
+
+Usage:
+  notation sign [reference] [flags]
+
+Flags:
+      --cert-file string        signing certificate file
+  -e, --expiry duration         expire duration
+  -h, --help                    help for sign
+  -k, --key string              signing key name
+      --key-file string         signing key file
+  -l, --local                   reference is a local file
```

Comment:
Can we explicitly claim this flag needs to be used with `-o` together?

Reply:
Second thought: We can remove both "-l" and "-o" flag for rc.1, because we don't have a complete use story implemented for signing local artifacts. It just works partially. We can add it back once we have a complete user story implemented.